tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -69,6 +68,7 @@ func TestRateLimiting_DefaultLessThanOverride(t *testing.T) {
 				deleteConfigOrFail(t, config, g, ctx)
 				util.AllowRuleSync(t)
 			}()
+			prom := prometheus.NewOrFail(t, ctx)
```

Comment:
this doesn't create a new pod, just a new port-foward to the same pod

Reply:
Thanks.. Added a config to restart prom pod